<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppletCentralWidget</class>
 <widget class="QMainWindow" name="AppletCentralWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1339</width>
    <height>907</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ilastik Classification Workflow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="VolumeEditorWidget" name="volumeEditorWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>99</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1339</width>
     <height>25</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionFitToScreen"/>
    <addaction name="actionToggleAllHuds"/>
    <addaction name="actionCenterAllImages"/>
    <addaction name="actionShowDebugPatches"/>
    <addaction name="separator"/>
    <addaction name="actionOnly_for_current_view"/>
    <addaction name="actionFitImage"/>
    <addaction name="actionToggleSelectedHud"/>
    <addaction name="actionCenterImage"/>
    <addaction name="actionReset_zoom"/>
    <addaction name="actionRubberBandZoom"/>
   </widget>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset>
     <normaloff>icons/document-open.png</normaloff>icons/document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open File</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFitToScreen">
   <property name="icon">
    <iconset>
     <normaloff>icons/view-fullscreen.png</normaloff>icons/view-fullscreen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Zoom to &amp;fit</string>
   </property>
   <property name="toolTip">
    <string>Fit all</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset>
     <normaloff>icons/process-stop.png</normaloff>icons/process-stop.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionShowDebugPatches">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Show Debug Patches</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionToggleAllHuds">
   <property name="text">
    <string>Toggle huds</string>
   </property>
   <property name="toolTip">
    <string>Hide or show all huds</string>
   </property>
  </action>
  <action name="actionOnly_for_current_view">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Only for selected view</string>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFor_all_axis">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>For all axis</string>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionToggleSelectedHud">
   <property name="text">
    <string>Toggle hud</string>
   </property>
   <property name="toolTip">
    <string>toggle selected hud</string>
   </property>
   <property name="shortcut">
    <string>H</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionCenterAllImages">
   <property name="text">
    <string>Center Images</string>
   </property>
  </action>
  <action name="actionCenterImage">
   <property name="text">
    <string>Center Image</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionFitImage">
   <property name="text">
    <string>Fit image</string>
   </property>
   <property name="toolTip">
    <string>fit image into view</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionReset_zoom">
   <property name="text">
    <string>Reset zoom</string>
   </property>
   <property name="toolTip">
    <string>reset zoom</string>
   </property>
   <property name="shortcut">
    <string>W</string>
   </property>
  </action>
  <action name="actionRubberBandZoom">
   <property name="text">
    <string>Rubber band zoom</string>
   </property>
   <property name="toolTip">
    <string>Zoom selected area</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionOpenStack">
   <property name="icon">
    <iconset>
     <normaloff>icons/document-open.png</normaloff>icons/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Open &amp;Stack</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VolumeEditorWidget</class>
   <extends>QWidget</extends>
   <header>volumina.volumeEditorWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
